keepassxc-browser/options/options.js

```diff
@@ -200,9 +200,22 @@ options.initGeneralSettings = async function() {
     });
 
     $('#configureCommands').addEventListener('click', function() {
+        if (isFirefox()) {
+            // Opening extension shortcuts is only supported with Firefox 137 and newer
+            if (getBrowserMajorVersion() >= 137) {
+                browser.commands.openShortcutSettings();
```

Comment:
Is there some way to test if `browser.commands` has the `openShortcutSettings` function? I don't necessarily like testing for version number, especially if Firefox variants don't implement it or have different version schemes.

Reply:
True, let's do that instead.

Reply:
It is needed. Older Firefox versions (for example Firefox ESR that's quite common) doesn't have this feature yet.